scripts/fix-colorings.py

```diff
@@ -86,4 +86,4 @@ def recurse(node):
     adjust_coloring_for_epiweeks(input_json)
 
     with open(args.output, 'w') as f:
-        json.dump(input_json, f, indent=2)
+        json.dump(input_json, f, indent=0)
```

Comment:
Since we already have [a convention in Augur of controlling indentation with an environment variable](https://github.com/nextstrain/augur/blob/0db0a419d0fa04121bb354389285dc448c67249b/augur/utils.py#L227) named `AUGUR_MINIFY_JSON `, we could maintain that interface here and allow users to override this default.

Reply:
Something like this?

```suggestion
        from augur.utils import write_json  # at the top of the file
        write_json(input_json, f)
```

I didn't know that Augur had that option until you pointed that out to me. I also learned that [the nextstrain-cli can pass this environment variable (and others) directly to the underlying execution platforms](https://github.com/nextstrain/cli/blob/0926451746605d5b7edd6cfcec0d8f82da8b5449/nextstrain/cli/hostenv.py#L18-L36).

If we decide to go with that, I would have to see how big the diffs get. Does having newlines impact the size of the diffs at all?

Reply:
That's pretty close! I think we'd also want to pass an argument to exclude the Augur version from the output like so:

```
write_json(input_json, f, include_version=False)
```

Do you want to try this and see how it works for you, @fanninpm?

Regarding diffs, do you mean when storing your Auspice JSONs in version control? If so, newlines will have an effect on diff sizes in the sense that a single-line Auspice JSON will look like a single new line of JSON each time you commit it.